pkg/scheduler/framework/plugins/defaultbinder/default_binder.go

```diff
@@ -55,15 +58,40 @@ func (pl *DefaultBinder) Bind(ctx context.Context, sub *appsapi.Subscription, ta
 	// use an ordered list
 	sort.Stable(targetClusters)
 
-	subCopy := sub.DeepCopy()
-	subCopy.Status.BindingClusters = targetClusters.BindingClusters
-	subCopy.Status.Replicas = targetClusters.Replicas
-	subCopy.Status.SpecHash = utils.HashSubscriptionSpec(&subCopy.Spec)
-	subCopy.Status.DesiredReleases = len(targetClusters.BindingClusters)
+	Status := sub.Status.DeepCopy()
```

Comment:
It is better not to get scheduler involved.

Reply:
> It is better not to get scheduler involved.

The scheduler will update the status of subs here, here I add a RetryOnConflict to UpdateStatus.